packages/react/src/components/FileUploader/Filename.js

```diff
@@ -36,7 +36,8 @@ function Filename({ iconDescription, status, invalid, ...rest }) {
         <CheckmarkFilled
           aria-label={iconDescription}
           className={`${prefix}--file-complete`}
-          {...rest}>
+          {...rest}
+          tabIndex={null}>
```

Comment:
Out of curiosity, what would the difference, if any, be between having tabIndex equal null or tabIndex equal to -1?

Reply:
When a prop is set to `null` it shouldn't render that prop on the component at all. Since it's an SVG and they don't support `tabIndex`, I figured I'd just remove it in that case

Reply:
I think the main use case of this would be a `disabled` prop where you would do something like `disabled={disabled ? true : null}`